manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -309,8 +309,16 @@ data:
           {{- end }}
         {{- end }}
           securityContext:
-            allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
-            privileged: {{ .Values.global.proxy.privileged }}
+          {{- if .Values.global.proxy.privileged }}
+            allowPrivilegeEscalation: true
+            privileged: true
+          {{- else if and (eq (annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode) `TPROXY`) (not .Values.istio_cni.enabled)}}
```

Comment:
Requiring the sidecar to be `privileged` is not acceptable.
If the `sysctl` command was really required (which it is really not), then `TPROXY`+original-src support should be implemented in Istio CNI only, and not support in non-Istio-CNI deployments.
This is a blocker and non-negotiable to me.